vault/policy_store.go

```diff
@@ -259,7 +259,8 @@ func (c *Core) setupPolicyStore(ctx context.Context) error {
 		return err
 	}
 
-	if c.ReplicationState().HasState(consts.ReplicationPerformanceSecondary | consts.ReplicationDRSecondary) {
+	if c.ReplicationState().HasState(consts.ReplicationPerformanceSecondary |
```

Comment:
ReplicationState() doesn't include information about perf standby status.  Use c.perfStandby instead.  Normally you need to ensure you hold a lock to access that field, but unseal holds the lock so you're ok here.

Reply:
Fixed 👍